examples/get-started/pure-js/google-maps/app.js

```diff
@@ -6,9 +6,11 @@ import {GeoJsonLayer, ArcLayer} from '@deck.gl/layers';
 const AIR_PORTS =
   'https://d2ad6b4ur7yvpq.cloudfront.net/naturalearth-3.3.0/ne_10m_airports.geojson';
 
+const GOOGLE_MAP_ID = 'f865292e93a85c6c';
```

Comment:
What is this magic string? Is it something that all users would want to use?

Reply:
It is defined on a per-user basis, they obtain from the Google cloud console. I've removed it an instead it is now provided via an environment variable, like we do with `GOOGLE_MAPS_API_KEY`.

Reply:
Is it just for styling? Is it possible to create a map without it?

Reply:
It is required to create a vector map, not just for styling. I have updated the README